core/server/data/schema/schema.js

```diff
@@ -453,6 +453,13 @@ module.exports = {
             defaultTo: 'pending',
             validations: {isIn: [['pending', 'submitting', 'submitted', 'failed']]}
         },
+        recipient_filter: {
+            type: 'string',
+            maxlength: 50,
+            nullable: false,
+            defaultTo: 'paid',
```

Comment:
What do you think about this default value? It felt the safest for me, and we need a default value when adding a new column if the column is not nullable (AFAIK?)

Reply:
We are going to run another migration to remove the default value